PROCESSORS.md

```diff
@@ -465,6 +465,38 @@ def unpivot(unpivot_fields, extra_keys, extra_value, resources=None):
   - `None` indicates operation should be done on all resources
   - The index of the resource in the package
 
+Examples:
```

Comment:
@anuveyatsu this is great but it would be nice to explain what you expect the output to look like - and showing that output at the bottom.

Reply:
:+1: for showing the actual output in the end.